### DISCUSSION TITLE
付款后专业版仍然提示到期续费，请解决

### DISCUSSION THREAD

**Discussion by szszszneuron** [getlantern/lantern-forum-cn#562] [General]
昨天付款后，登录专业版，仍然提醒专业版到期需要续费。
当时我专业版的邮箱是： szneuron@126.com
付款凭证见下图，请帮忙解决一下，或者退款我重来一下。谢谢！
![微信图片_20240613082454](https://github.com/getlantern/lantern-forum-cn/assets/172555438/4509b84e-a059-4455-b34f-54947dc7cba9)

**Comment by MaxMKMax** [comment]
@szszszneuron 您好. 已经帮您提交问题了. 过几天请查看邮箱或这个帖子. 
我们这里可以帮您手动升级. 你不介意的话可以等到下周二吗? 负责人不在.
谢谢您支持蓝灯!

**Comment by szszszneuron** [comment]
好的，谢谢！那就下周二帮我升级一下吧。用蓝灯好几年了。




---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年6月13日 17:58 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 付款后专业版仍然提示到期续费，请解决 (Discussion #562) |

@szszszneuron 您好. 已经帮您提交问题了. 过几天请查看邮箱或这个帖子.
我们这里可以帮您手动升级. 你不介意的话可以等到下周二吗? 负责人不在.
谢谢您支持蓝灯!

**Comment by szszszneuron** [comment]
请记得明天帮我处理下一下我这个问题，谢谢！






---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年6月13日 20:42 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | 回复： [getlantern/lantern-forum-cn] 付款后专业版仍然提示到期续费，请解决 (Discussion #562) |
好的，谢谢！那就下周二帮我升级一下吧。用蓝灯好几年了。




---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年6月13日 17:58 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 付款后专业版仍然提示到期续费，请解决 (Discussion #562) |

@szszszneuron 您好. 已经帮您提交问题了. 过几天请查看邮箱或这个帖子.
我们这里可以帮您手动升级. 你不介意的话可以等到下周二吗? 负责人不在.
谢谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@szszszneuron 您好. 我这里和您跟进一下. 您的问题解决了吗?

**Comment by szszszneuron** [comment]
您好，问题已解决，感谢！






---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年7月16日 22:53 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 付款后专业版仍然提示到期续费，请解决 (Discussion #562) |

@szszszneuron 您好. 我这里和您跟进一下. 您的问题解决了吗?

**Reply by MaxMKMax** [reply]
@szszszneuron 您好. 你客气了. 可以服务您是我们的荣幸. 谢谢您支持蓝灯.

**Comment by szszszneuron** [comment]
最近你们这个基本上连不上呀，都是这种界面，怎么回事哦
感觉现在越来越不好用了。
 







---- 回复的原邮件 ----
| 发件人 | Mike ***@***.***> |
| 发送日期 | 2024年7月17日 23:08 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***>,
***@***.***> |
| 主题 | Re: [getlantern/lantern-forum-cn] 付款后专业版仍然提示到期续费，请解决 (Discussion #562) |

@szszszneuron 您好. 你客气了. 可以服务您是我们的荣幸. 谢谢您支持蓝灯.
